.github/workflows/build-appimage.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+set -e
+
+export ARCH="$(uname -m)"
+export APPIMAGE_EXTRACT_AND_RUN=1
+export VERSION=$GITHUB_REF_NAME
+
+APPIMAGETOOL="https://github.com/AppImage/appimagetool/releases/download/continuous/appimagetool-x86_64.AppImage"
+UPINFO="gh-releases-zsync|$GITHUB_REPOSITORY_OWNER|UniversalMediaServer|latest|*$ARCH.AppImage.zsync"
+
+cd target
+tar fx ./*.tar.gz
+
+cd ./ums-$VERSION
+cp ./web/react-client/icon-256.png ./ums.png
+ln -s ./UMS.png ./.DirIcon
+echo '[Desktop Entry]
+Version=1.0
+Name=Universal Media Server
+Comment=A DLNA-compliant UPnP Media Server.
+Exec=UMS.sh
+Icon=ums
+Terminal=false
+Type=Application
+Categories=Java;AudioVideo;' > ums.desktop
+ln -s ./UMS.sh ./AppRun
+cd ..
+
+wget "$APPIMAGETOOL" -O ./appimagetool
+
+chmod +x ./appimagetool
+
+echo "$(ls)"
+
+mv ./ums* ./ums.AppDir
```

Comment:
change this to `mv ./ums-$VERSION ./ums.AppDir`

Note I'm assuming that's the right directory that the script used to download and extract before, that is the dir where `UMS.sh` is located.

Reply:
Thanks for the help, it's working now